spec/models/order_spec.rb

```diff
@@ -0,0 +1,61 @@
+require "rails_helper"
+
+RSpec.describe Order, type: :model do
+  describe "validates" do
+    it do
+      is_expected.to validate_presence_of(:phone)
+      is_expected.to validate_presence_of(:address)
+
+      is_expected.to define_enum_for(:status).with_values(%i(wait_confirm delivering completed canceled))
+    end
+  end
+
+  describe "associations" do
+    it do
+      is_expected.to have_many(:order_items)
+      is_expected.to have_many(:products).through(:order_items)
+      is_expected.to belong_to(:user)
+    end
+  end
+
+  describe "callback" do
+    let!(:product) { FactoryBot.create(:product, price: 20) }
```

Comment:
như thế này thì sao nhỉ?
```suggestion
    let(:product) { FactoryBot.create(:product, price: 20) }
```

Reply:
@taylp-2865 do ở let phía dưới e dùng tới á anh nên e dùng ! luôn